TYPO3.Neos/Resources/Public/JavaScript/Content/Components/TargetWorkspaceSelector.js

```diff
@@ -22,20 +22,20 @@ define(
       classNameBindings: ['open:neos-open'],
       template: Ember.Handlebars.compile(template),
 
-      controller: TargetWorkspaceController,
-      nodeSelection: NodeSelection,
-      publishableNodes: PublishableNodes,
+			_controller: TargetWorkspaceController,
```

Comment:
@aertmann why do you rename controller here? I think this way it's no longer accessible in TargetWorkspaceSelector.html `content=targetWorkspaces`